project/src/components/main/main.tsx

```diff
@@ -0,0 +1,66 @@
+import PlaceCard from '../place-card/place-card';
+import Locations from '../locations/locations';
+import Map from '../map/map';
+import Header from '../header/header';
+
+type MainProps = {
+  placesCount: number;
+}
+
+function Main({placesCount}: MainProps): JSX.Element {
+  return (
+    <div className="page page--gray page--main">
+      {<Header />}
+      <main className="page__main page__main--index">
+        <h1 className="visually-hidden">Cities</h1>
+        <div className="tabs">
+          <section className="locations container">
+            {<Locations/>}
+          </section>
+        </div>
+        <div className="cities">
+          <div className="cities__places-container container">
+            <section className="cities__places places">
+              <h2 className="visually-hidden">Places</h2>
+              <b className="places__found">{placesCount} places to stay in Amsterdam</b>
+              <form className="places__sorting" action="#" method="get">
+                <span className="places__sorting-caption">Sort by</span>
+                <span className="places__sorting-type" tabIndex={0}>
+                  Popular
+                  <svg className="places__sorting-arrow" width={7} height={4}>
+                    <use xlinkHref="#icon-arrow-select" />
+                  </svg>
+                </span>
+                <ul className="places__options places__options--custom places__options--opened">
+                  <li
+                    className="places__option places__option--active"
+                    tabIndex={0}
+                  >
+                    Popular
+                  </li>
+                  <li className="places__option" tabIndex={0}>
+                    Price: low to high
+                  </li>
+                  <li className="places__option" tabIndex={0}>
+                    Price: high to low
+                  </li>
+                  <li className="places__option" tabIndex={0}>
+                    Top rated first
+                  </li>
+                </ul>
+              </form>
+              <div className="cities__places-list places__list tabs__content">
+                {Array.from({length: placesCount}, PlaceCard)}
```

Comment:
```suggestion
                {Array.from({length: placesCount}, () => <PlaceCard />)}
```
лучше явно укажи что компонент, а то можно запутаться